powermon/commands/command.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, code: str, commandtype: str, outputs: list[AbstractOutput], t
         log.debug(self)
 
     def build_result(self, raw_response=None, protocol=None) -> Result:
-        log.debug("build_result: code:%s, command_definition:%s", self.code, self.command_definition)
+        log.debug(f"build_result: code:{self.code}, command_definition:{self.command_definition}")
```

Comment:
Ive been trying to move f-strings to lazy expansion in debug messages (as per pylint recommendations)

Reply:
It wasn't printing properly during an exception being thrown so it wasn't very helpful. I can switch it back, should work if there isn't an exception.